### ISSUE TITLE
Whitespaces in bind-hooks

### ISSUE THREAD

**Issue by parazyd** [dyne/Tomb#222]
While trying to mount my tomb with bind hooks, I discovered an error in parsing. Basically, if a path has spaces in it, it is not parsed correctly. While creating a symlink to the problematic path, it managed to work as a hack for tomb opening and mounting, but it does not help when closing the same tomb.

Here's the problematic entry in bind-hooks:
`palemoon		.pro`
Where .pro is a symlink
`.pro -> .moonchild productions/pale moon/5xe6rqho.default`

Issuing `tomb close all` or `tomb slam all` gives the following output, respectively:
```
tomb  .  Closing tomb [OMMITED mounted on /media/OMMITED  
tomb  .  Closing tomb bind hook: OMMITED BECAUSE MUH PRIVACY  
tomb  .  Closing tomb bind hook: OMMITED BECAUSE MUH PRIVACY  
tomb  .  Closing tomb bind hook: OMMITED BECAUSE MUH PRIVACY  
tomb  .  Closing tomb bind hook: OMMITED BECAUSE MUH PRIVACY  
umount_tomb:72: failed to compile regex: Invalid range end  
device-mapper: remove ioctl on tomb.OMMITED BECAUSE MUH PRIVACYloop0 failed: Device or resource busy  
device-mapper: remove ioctl on tomb.OMMITED-BECAUSE-MUH-PRIVACY.loop0 failed: Device or resource busy  
device-mapper: remove ioctl on tomb.OMMITED-BECAUSE-MUH-PRIVACY.loop0 failed: Device or resource busy  
device-mapper: remove ioctl on tomb.OMMITED-BECAUSE-MUH-PRIVACY.loop0 failed: Device or resource busy  
device-mapper: remove ioctl on tomb.OMMITED-BECAUSE-MUH-PRIVACY.loop0 failed: Device or resource busy  
device-mapper: remove ioctl on tomb.OMMITED-BECAUSE-MUH-PRIVACY.loop0 failed: Device or resource busy  
device-mapper: remove ioctl on tomb.OMMITED-BECAUSE-MUH-PRIVACY.loop0 failed: Device or resource busy  
device-mapper: remove ioctl on tomb.OMMITED-BECAUSE-MUH-PRIVACY.loop0 failed: Device or resource busy  
device-mapper: remove ioctl on tomb.OMMITED-BECAUSE-MUH-PRIVACY.loop0 failed: Device or resource busy  
device-mapper: remove ioctl on tomb.OMMITED-BECAUSE-MUH-PRIVACY.loop0 failed: Device or resource busy  
device-mapper: remove ioctl on tomb.OMMITED-BECAUSE-MUH-PRIVACY.loop0 failed: Device or resource busy  
device-mapper: remove ioctl on tomb.OMMITED-BECAUSE-MUH-PRIVACY.loop0 failed: Device or resource busy  
device-mapper: remove ioctl on tomb.OMMITED-BECAUSE-MUH-PRIVACY.loop0 failed: Device or resource busy  
device-mapper: remove ioctl on tomb.OMMITED-BECAUSE-MUH-PRIVACY.loop0 failed: Device or resource busy  
device-mapper: remove ioctl on tomb.OMMITED-BECAUSE-MUH-PRIVACY.loop0 failed: Device or resource busy  
device-mapper: remove ioctl on tomb.OMMITED-BECAUSE-MUH-PRIVACY.loop0 failed: Device or resource busy  
device-mapper: remove ioctl on tomb.OMMITED-BECAUSE-MUH-PRIVACY.loop0 failed: Device or resource busy  
device-mapper: remove ioctl on tomb.OMMITED-BECAUSE-MUH-PRIVACY.loop0 failed: Device or resource busy  
device-mapper: remove ioctl on tomb.OMMITED-BECAUSE-MUH-PRIVACY.loop0 failed: Device or resource busy  
device-mapper: remove ioctl on tomb.OMMITED-BECAUSE-MUH-PRIVACY.loop0 failed: Device or resource busy  
device-mapper: remove ioctl on tomb.OMMITED-BECAUSE-MUH-PRIVACY.loop0 failed: Device or resource busy  
device-mapper: remove ioctl on tomb.OMMITED-BECAUSE-MUH-PRIVACY.loop0 failed: Device or resource busy  
device-mapper: remove ioctl on tomb.OMMITED-BECAUSE-MUH-PRIVACY.loop0 failed: Device or resource busy  
device-mapper: remove ioctl on tomb.OMMITED-BECAUSE-MUH-PRIVACY.loop0 failed: Device or resource busy  
device-mapper: remove ioctl on tomb.OMMITED-BECAUSE-MUH-PRIVACY.loop0 failed: Device or resource busy  
Device tomb.OMMITED-BECAUSE-MUH-PRIVACY.loop0 is still in use.  
tomb [E] Error occurred in cryptsetup luksClose tomb.OMMITED-BECAUSE-MUH-PRIVACY.loop0
```
and
```
tomb (*) Slamming tomb ext4 mounted on /home/username/.moonchild  
tomb  .  Kill all processes busy inside the tomb.  
umount: /home/username/.moonchild: mountpoint not found  
tomb [E] Tomb is busy, cannot umount!
```

Perhaps this issue is related: https://github.com/dyne/Tomb/issues/220  
Also using Arch.

**Comment by jaromil** [comment]
thanks a lot!!

**Comment by zhogano** [comment]
How can I implement bind-hooks if there is a space character in the path relative to $HOME? I enclosed the path names with "" but it did not work. Then I tried using symlink. Tomb opened without problem, but I was unable to close it with `tomb close` and `tomb slam`. I am using Tomb 2.9.0. I can not remove the space character from the folder name, because it was created by a software and it won't work if I change it.
